### ISSUE TITLE
Landing page conversion

### ISSUE THREAD

**Issue by dkuku** [AgileVentures/agile-ventures-website-react-front-end#23]
We are still importing bootstrap 3 in the public/index.html file and it's used on the landing page.
This may conflict with semantic ui - some class names are the same.

**Comment by mattwr18** [comment]
yes @dkuku we need to purge our project of bootstrap, in my opinion.
